src/CMakeLists.txt

```
@@ -124,6 +124,11 @@ if (TILES)
         target_link_libraries(cataclysm-tiles-common PUBLIC backtrace)
     endif ()
 
+    if (USE_TRACY)
+        target_link_libraries(cataclysm-tiles PUBLIC TracyClient)
+        target_include_directories(cataclysm-tiles-common SYSTEM PUBLIC ${tracy_SOURCE_DIR}/public)
```

Needs define to work
```suggestion
        target_include_directories(cataclysm-tiles-common SYSTEM PUBLIC ${tracy_SOURCE_DIR}/public)
        target_compile_definitions(cataclysm-tiles-common PUBLIC USE_TRACY)
```